components/toolbar/brave_toolbar_model_impl.cc

```diff
@@ -32,5 +37,21 @@ base::string16 BraveToolbarModelImpl::GetURLForDisplay() const {
     formatted_text.replace(0, original_scheme_part.length(),
                             replacement_scheme_part);
   }
```

Comment:
Previously when I did the security review for using `GetURLForDisplay` to map `brave://` to `chrome://` in the URL bar, I assumed the returned URL was not being used for any security checks. However it turns out that assumption was wrong (https://github.com/brave/brave-browser/issues/2631 - sorry you can't see the details yet! can share in Slack).

Reply:
actually ignore my above comment, i got https://github.com/brave/brave-browser/issues/213 mixed up with https://github.com/brave/brave-browser/issues/1458